includes/theme.php

```diff
@@ -289,7 +289,16 @@ function ap_post_actions( $_post = null ) {
 		$actions[] = array( 'header' => true );
 	}
 
-	if ( ap_user_can_delete_post( $_post->ID ) ) {
+	$answers = get_posts(
+		array(
+			'post_type'   => 'answer',
+			'post_status' => 'publish',
+			'post_parent' => $_post->ID,
+			'showposts'   => -1,
```

Comment:
@mebishalnapit Replace this query with [wp_count_posts](https://developer.wordpress.org/reference/functions/wp_count_posts/)

Reply:
Updated to use `wp_count_posts` function instead.